### PR TITLE
Adding checks for additional possible infra failures

### PR DESCRIPTION
**Pull request by ewolinetz**
cc @stevekuznetsov @wking

### CODE REVIEW

File: pkg/api/config_test.go [hunk 1]
```diff
@@ -75,7 +75,7 @@ func TestValidateTests(t *testing.T) {
 			id: "test without `commands`",
 			tests: []TestStepConfiguration{
 				{
-					As:                         "test",
+					As: "test",
```
Comment (wking) [inline pkg/api/config_test.go]: Looks like you washed this through an older `gofmt`?

**Comment by wking** [comment]
what do we use this for?  Is this cache-busting?

**Comment by ewolinetz** [comment]
@wking basically... We had noticed on one of my jobs that the image builds had failed due to `yum` and retesting would immediately fail because of the pre-existing namespace

**Pull request merged**